### ISSUE TITLE
Focus issue after selecting a ToC element

### ISSUE THREAD

**Issue by globau** [mdn/fred#523]
When selecting a section in ToC the content moves but focus stays in ToC - it should switch to content section.

> Task: **Navigate to "Examples" from the sidebar and play the CSS code under it.**
>
> **When I select a button or link, nothing happens**
> When I navigate to and select Examples from the sidebar, the focus does not move to that section, it just stays in the sidebar. It would be helpful if pressing the link from the sidebar moved the focus to that section so I don't have to scroll all the way to that section.

**Comment by caugner** [comment]
I cannot reproduce the issue:

1. Go to https://fred.review.mdn.allizom.net/en-US/docs/Web/API/Event/target
2. Tab to an entry in the ToC
3. Select the entry (either click or press <kbd>Enter</kbd>)
4. Focus switched to content (i.e. tabbing continues in selected heading)

Maybe I'm missing something.

**Comment by caugner** [comment]
@globau A colleague with NVDA couldn't reproduce the issue either, so I'd suggest to close as "can't repro".

**Comment by globau** [comment]
Unable to reproduce - closing.
